helm/logging-operator/templates/deployment.yaml

```diff
@@ -45,6 +45,10 @@ spec:
           {{- with .Values.securityContext.container }}
             {{- . | toYaml | nindent 10 }}
           {{- end }}
```

Comment:
Do we need that for object-storage operator as well ?

Reply:
I couldn't say for sure, that's not part of this PR. But that looks like a good idea yes.

Reply:
Considering I copied this pod monitor I think so 😅